recipes/umongo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "umongo" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/Scille/umongo/archive/refs/tags/{{ version }}.tar.gz"
```

Comment:
I'm pretty sure the auto updating bot will break with these `"`. We also don't need them, can you remove them?